zingolib/src/lightclient/test_features.rs

```
@@ -85,4 +82,15 @@ impl LightClient {
             }
         }
     }
+
+    /// Creates a [`zcash_client_backend::zip321::TransactionRequest`] from rust primitives for simplified test writing.
+    pub fn transaction_request_from_send_inputs(
+        &self,
+        address_amount_memo_tuples: Vec<(&str, u64, Option<&str>)>,
+    ) -> TransactionRequest {
+        let receivers =
+            receivers_from_send_inputs(address_amount_memo_tuples, &self.config().chain);
+        transaction_request_from_receivers(receivers)
+            .expect("should be able to create a transaction request as receivers are valid.")
```

can we return an error value instead of expecting? its possible to get an error here.

---

valid receivers can still result in invalid request in some cases.

---

I dont have an issue with test only functions containing expects. if for example there were too many payments in a test, then a panic here would be enough information to correct the test. anyways I have added the return of the zip321 error for you